docs/integracion-continua.md

```diff
@@ -0,0 +1,15 @@
+# Integración Continua     
+Para la búsqueda me he centrado en aquellos sistemas de integración continua que se puedan usar con el leguaje del proyecto, que estén integrandos en la nube y que se puedan usar en GitHub. Dentro de estos he seleccionado para usar en el proyecto aquellos que sean gratuitos, que no se excedan en los permisos para vincularlos con GitHub y que no demanden ciertos datos personales como puede ser el número de tarjeta.   
+
+## **Búsqueda de CI:**    
+En la búsqueda de CI he encontrado: [AppVeyor](https://www.appveyor.com/), [Cirrus](https://github.com/apps/cirrus-ci) y las propias [Github Actions](https://docs.github.com/es/actions)    
+
+- **AppVeyor:** Es una plataforma de integración usada para construir y testear proyectos en GitHub y otros sitios como GitLab y BitBucket. No se ha seleccionado ya que pide acceso a las organizaciones en las que estés.    
+
+- **Cirrus:** Es otra plataforma de integración que es tratada como una aplicación de GitHub, que es la forma oficial y recomendada de integrarse con GitHub ya que ofrecen permisos mucho más granulares para acceder a los datos y además, no actuan como usuarios autentificados como ocurre con las OAuth App.   
```

Comment:
Lo importante no es esto y no tiene relevancia. Es el uso de Checks API, que además es el que permite que los tests que yo he hecho detecten que se han usado dos sistemas de CI. Esto os lo dije en clase, y además está en el guión. Sé que no tomáis notas en clase no tengo muy claro por qué. Si las hubiérais tomado y/o mirado atentamente el guión, os habríais dado cuenta de lo relevante de que sean "aplicación de GitHub", que es el hecho de que automáticamente use el Checks API

Reply:
Aaah disculpe, lo añado :ok_hand:

Reply:
Luis, que no se trata de que lo añadas o no. Se trata de que miréis qué es el checks API. Me da igual lo que añadas, lo que quiero es que aprendáis una seria de cosas que os van a ser útiles para vuestro trabajo.